SPECS/iptables/ip6save

```diff
@@ -1,13 +1,13 @@
 # init
 *filter
-:INPUT DROP [0:0]
-:FORWARD DROP [0:0]
-:OUTPUT DROP [0:0]
-# Allow local-only connections
+:INPUT ACCEPT [0:0]
+:FORWARD ACCEPT [0:0]
+:OUTPUT ACCEPT [0:0]
+-A INPUT -m state --state RELATED,ESTABLISHED -j ACCEPT
+-A INPUT -p ipv6-icmp -j ACCEPT
 -A INPUT -i lo -j ACCEPT
--A INPUT -m conntrack --ctstate RELATED,ESTABLISHED -j ACCEPT
-#keep commented till upgrade issues are sorted
-#-A INPUT -j LOG --log-prefix "FIREWALL:INPUT "
--A INPUT -p tcp -m tcp --dport 22 -j ACCEPT
--A OUTPUT -j ACCEPT
+-A INPUT -p tcp -m state --state NEW -m tcp --dport 22 -j ACCEPT
+-A INPUT -d fe80::/64 -p udp -m udp --dport 546 -m state --state NEW -j ACCEPT
```

Comment:
This rule isn't in the ipv4 version. What is this supposed to be?

Reply:
Nevermind; this seems to be DHCP. Shouldn't this be allowed in the IPv4 version, too?

Reply:
Two forums (shown below) helped me with this same question. TLDR "DHCP uses packet filters and these tie into the IP stack before the firewall." "The ISC DHCP client uses "raw sockets" to bypass iptables so even a "drop everything" firewall can configure it's network stack". The IPv6 rule accepts DHCPv6 traffic which is different for IPv4. However, the rule isn't necessary if you use stateless autoconfiguration, or statically configure your machines (https://unix.stackexchange.com/questions/452880/what-are-the-essential-iptables-rules-for-ipv6-to-work-properly) . 

Sources
https://forums.centos.org/viewtopic.php?t=8728 
https://www.linuxquestions.org/questions/linux-security-4/bypass-of-iptables-by-internal-programs-946035/